### PR TITLE
fix: width of price impact speedbump

### PR DESCRIPTION
**Pull request by just-toby**
adjust the max-width of the price impact speed bump content

https://www.notion.so/uniswaplabs/Weird-UI-state-of-warning-modal-746px-52f715bda79c466d89e5b08d374f10b5?pvs=4

### test plan
Screenshot 2023-03-13 at 12 40 34 PM

### CODE REVIEW

File: src/components/Swap/Summary/index.tsx [hunk 1]
```diff
@@ -287,23 +287,21 @@ export function SummaryDialog(props: SummaryDialogProps) {
   }, [ackPriceImpact, props.impact, showSpeedbump])
 
   return (
-    <>
+    <Column style={{ minWidth: isPageCentered ? Math.min(400, width) : 'auto', height: '100%' }} ref={setBoundary}>
```
Comment (vm) [inline src/components/Swap/Summary/index.tsx:290]: could something like this work?
```ts
{{ minWidth: isPageCentered ? 400 : 'auto', width: isPageCentered: '100%': undefined, height: '100%' }}
```

Comment (vm) [reply src/components/Swap/Summary/index.tsx:290]: also, is the 400 here related to the 420 width on speedbump?

File: src/components/Swap/Toolbar/ToolbarContext.tsx [hunk 2]
```diff
@@ -16,12 +18,19 @@ export function Provider({ children }: PropsWithChildren) {
   const onToggleOpen = () => setOpen((open) => !open)
   const collapse = () => setOpen(false)
 
+  const {
```
[the remark this answers is not in the document]
Comment (just-toby) [reply src/components/Swap/Toolbar/ToolbarContext.tsx:21]: yes sorry i meant to split this out - will send a different PR now

File: src/components/Swap/Summary/index.tsx (hunk 1 above)
Comment (just-toby) [reply src/components/Swap/Summary/index.tsx:290]: nope - this is just the min width of a page centered dialog. i've updated to name this value so it's more understandable.

**Review by vm — APPROVED** [review]

**Pull request merged**